### DISCUSSION TITLE
Is filtering by multiple transaction types supported?

### DISCUSSION THREAD

**Discussion by hanshsieh** [firefly-iii/firefly-iii#7635] [Firefly III] [answered]
Some APIs support filtering the transactions with transaction type. For example,
```
/v1/accounts/{id}/transactions
/v1/categories/{id}/transactions
```
These APIs have a `type` parameter to specify the transaction type. Does it support filtering with multiple transaction types? For example, filtering with both "deposit" and "withdrawal".
I have tried several combinations, but none of them work.
```
type=withdrawal,deposit
type=withdrawal&type=deposit
type[]=withdrawal&type[]=deposit
```

**Comment by JC5** [comment, marked as answer]
No, you'll have to make two calls at the moment.

**Reply by hanshsieh** [reply]
I see. Thanks.
